data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
This is a new functionality, doesn't matter which data pack version you have. You will need to change it !! Guys I think it is better to start a discussion about this in OTLand, every time someone answers here all participants are notified.